tests/Unit/inc/Engine/CriticalPath/CriticalCSSSubscriber/maybeGenerateCpcssMobile.php

```diff
@@ -33,7 +35,8 @@ public function setUp() {
 			$options,
 			null,
 		] );
-		$this->subscriber   = new CriticalCSSSubscriber( $this->critical_css, $options, null );
+		$this->processor_service = Mockery::mock( ProcessorService::class );
```

Comment:
@crystinutzaa Are we mocking methods on the `ProcessorService`? If no, then we don't need to have this as a property on the test class.

Reply:
I think this is resolved here: https://github.com/wp-media/wp-rocket/commit/3180e6571c892ccc5173782186766c1c0ea44b2e